src/test/scala/is/hail/TestUtils.scala

```diff
@@ -115,4 +118,7 @@ object TestUtils {
     }
     sb.result()
   }
+  
+  def fileByteEquality(file1: String, file2: String): Boolean =
```

Comment:
I think we tend to prefer verbs clauses for method names so I have a slight preference for `filesHaveSameBytes` or something like that.